lib/utils/idleState.ts

```diff
@@ -83,22 +84,23 @@ class IdleState {
   observers?: Observers;
   setState?: (state: 'active' | 'idle') => void;
 
-  constructor(timeout) {
+  constructor(timeout) {  
     this.timeout = timeout;
-    this[stateProperty] = STATE_ACTIVE;
+    this[stateProperty] = STATE_IDLE;
```

Comment:
If you are keen on a deep clean-up, I guess you can also remove the `stateProperty`.

Reply:
✅   Thanks for the review Josh 🙇‍♀️ .  The changes have been made.